seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/Firefox.scala

```diff
@@ -33,10 +36,21 @@ object Firefox extends SeleniumBrowser {
         |    oldErr.apply(console, arguments);
         |  };
         |  this.scalajsPopHijackedConsoleLog = function () {
-        |    var log = console_log_hijack;
-        |    if (log.length != 0)
-        |      console_log_hijack = [];
-        |    return log;
+        |    if (console_log_hijack.length == 0) {
+        |      return console_log_hijack;
+        |    } else {
+        |      var log = [];
+        |      while (currentLogIndex < console_log_hijack.length &&
+        |          log.length < 1024) {
+        |        log.push(console_log_hijack[currentLogIndex]);
+        |        currentLogIndex++;
+        |      }
+        |      if (console_log_hijack.length == currentLogIndex) {
+        |        console_log_hijack = [];
+        |        currentLogIndex = 0;
+        |      }
```

Comment:
Why not remove all the entries from `console_log_hijack` that are sent, rather than maintaining a "global" `currentLogIndex`?